Formula/root.rb

```diff
@@ -14,12 +14,15 @@ class Root < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "davix"
   depends_on "fftw"
   depends_on "gcc" # for gfortran.
   depends_on "graphviz"
   depends_on "gsl"
+  depends_on "lz4"
```

Comment:
I see you set `-Ddavix=ON` but no `-Dlz4=ON`. Is there a reason?

Reply:
LZ4 is not an optional dependency anymore, it is now the defult compression algorithm for ROOT.
So if we don't depend on it, ROOT will build the bundled version. The dependency is to avoid that.